tests/regression.ml

```diff
@@ -75,6 +75,39 @@ let test_distinguished_name () =
   Alcotest.(check check_dn "complex subject is good"
               expected (Certificate.subject crt))
 
+let test_distinguished_name_pp () =
+  let module Dn = struct
+    include Distinguished_name
+    let cn s = Relative_distinguished_name.singleton (CN s)
+    let o s = Relative_distinguished_name.singleton (O s)
+    let initials s = Relative_distinguished_name.singleton (Initials s)
+    let (+) = Relative_distinguished_name.union
+  end in
+  let dn1 = "DN1", Dn.[o "Blanc";
+                       cn "John Doe" + initials "J.D." + initials "N.N."] in
+  let dn2 = "DN2", Dn.[o "Escapist"; cn "# 2"; cn " \"+,;/<>\\  "] in
```

Comment:
wouldn't `o " Escapist"` as well be nice for testing (iirc a leading space should lead to a quoted string)?

Reply:
There is already leading (and training) space in the last cn component, but I can put one there, as well.